pkg/capabilities/consensus/ocr3/aggregators/reduce_aggregator.go

```diff
@@ -185,6 +180,29 @@ func (a *reduceAggregator) Aggregate(lggr logger.Logger, previousOutcome *types.
 	}, nil
 }
 
+func (a *reduceAggregator) shouldReport(lggr logger.Logger, field AggregationField, singleValue values.Value, currentState *map[string]values.Value) (bool, error) {
+	if field.DeviationType == DEVIATION_TYPE_NONE {
+		return true, nil
+	}
```

Comment:
@agparadiso This isn't correct: this means that if we have a struct where a single field doesn't have a deviation defined on it, we'll always report.

That means we'll always report for our existing struct, which looks like:
- Price <- 5 % deviation
- Timestamp <- 300 s deviation
- FeedID <- no deviation

If the deviation type is none, we should instead look at the previous entry and only report if it's different.

Reply:
addressed on the last commit 👍🏼

Reply:
> If the deviation type is none, we should instead look at the previous entry and only report if it's different.

@cedric-cordenier we added a new `deviationType` to cover this behavior, called "any". Basically if the value ever changes, send a report.

This is different from "none", which is never send a report if this value changes. (the default behavior)

Reply:
Sounds good!